From 70c3e0c33e9f9285d1e618bde438991fee9a3c38 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Sat, 20 Jan 2024 01:09:31 +0000
Subject: [PATCH] Update elements/components/wayland-protocols.bst to 1.33-0

Updates elements/components/wayland-protocols.bst to 1.33-0 (5434607)
Previous version was 1.32-0 (681c33c)
---
 elements/components/wayland-protocols.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/wayland-protocols.bst b/elements/components/wayland-protocols.bst
index 2bc106756a1..3b0cdfc8f38 100644
--- a/elements/components/wayland-protocols.bst
+++ b/elements/components/wayland-protocols.bst
@@ -11,4 +11,4 @@ sources:
 - kind: git_repo
   url: freedesktop:wayland/wayland-protocols.git
   track: '*.*'
-  ref: 1.32-0-g681c33c8547d6aefe24455ba2bffe1c5ae11fee5
+  ref: 1.33-0-g54346071a5f211f2c482889f2c8ee3b5ecda63ab
-- 
GitLab

